api/Console.json

```
@@ -851,20 +872,23 @@
               "notes": "Alias for <code>console.log</code>"
             },
             "opera": {
-              "version_added": true
+              "version_added": "10.5"
+            },
+            "opera_android": {
+              "version_added": "11"
             },
             "safari": {
               "version_added": true,
```

```suggestion
              "version_added": "3",
```

This is api.Console.info which you had tested as Safari ≤3. Given that `console.log` is marked as supported from Safari 3, I suggest doing the same here. It's not likely this was introduced before `console.log`, and if they both were introduced earlier, there's very little harm in being wrong between Safari 1 and 3.

---

Good catch!  I should've noticed when the `console` API was implemented, haha -- certainly saves the trouble of testing in Safari 1-2!